### DISCUSSION TITLE
Transliteration filter

### DISCUSSION THREAD

**Discussion by urbanit** [OrchardCMS/OrchardCore#10748] [Ideas]
* Should we provide a transliteration filter?
* Should we include it as slugify filter option?
* How should we implement specific culture mappings? Crowdin?

Related Issues:
[4151](https://github.com/OrchardCMS/OrchardCore/issues/4151)
[4874](https://github.com/OrchardCMS/OrchardCore/issues/4874)

**Comment by hishamco** [comment]
I'm investigating ICU4N ..

- We should provide a transliteration filter
- We could include it as an option to slugify filter as Seb mentioned earlier

I think this should be related to content localization or localization in OC and could be utilized everywhere else

**Comment by urbanit** [comment]
Related PR: https://github.com/OrchardCMS/OrchardCore/pull/19009
